meta-mender-qemu/recipes-mender/mender-client/mender_%.bbappend

```diff
@@ -1,4 +1,18 @@
-require mender-client-test-files.inc
```

Comment:
I think this is wrong (I am unsure if this _was_ correct in the original PR).

Either we keep the `require` and the changes in `mender-client-test-files.inc`, or we remove completely the inc file and keep the changes here in the `mender_%.bbappend`. --> IOW check that the inc file is only included from this bbapend file.